examples/installation/docker-compose.yml

```diff
@@ -0,0 +1,217 @@
+name: nussknacker
+
+services:
+
+  nginx:
+    image: nginx:1.26.0-alpine
+    restart: unless-stopped
+    ports:
+      - 8080:8080
+    depends_on:
+      grafana:
+        condition: service_healthy
+      designer:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "service", "nginx", "status"]
+      interval: 10s
+      retries: 10
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./nginx/additional:/etc/nginx/additional
+
+  ### Nussknacker designer service
+
+  designer:
+    image: touk/nussknacker:1.14.0_scala-2.12
+    restart: unless-stopped
+    environment:
+      DB_URL: "jdbc:postgresql://postgres:5432/nu-db"
+      DB_DRIVER: "org.postgresql.Driver"
+      DB_USER: "nu"
+      DB_PASSWORD: "nupassword"
+      NUSSKNACKER_LOG_LEVEL: "INFO"
+      KAFKA_ADDRESS: "kafka:9092"
+      SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
+      INFLUXDB_URL: "http://influxdb:8086"
+      FLINK_REST_URL: "http://flink-jobmanager:8081"
+      FLINK_QUERYABLE_STATE_PROXY_URL: "flink-taskmanager:9069"
+      JDK_JAVA_OPTIONS: "-Xmx1024M"
+      USAGE_REPORTS_SOURCE: "example-installation-docker-compose"
+    depends_on:
+      postgres:
+        condition: service_healthy
+      grafana:
+        condition: service_healthy
+      kafka:
+        condition: service_healthy
+      schema-registry:
+        condition: service_healthy
+      flink-jobmanager:
+        condition: service_healthy
+      flink-taskmanager:
+        condition: service_started
+    healthcheck:
+      test: [ "CMD-SHELL", "curl localhost:8080/api/app/healthCheck" ]
+      interval: 10s
+      retries: 10
+    volumes_from:
+      - flink-jobmanager
+
+  postgres:
+    image: postgres:13
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: "nu-db"
+      POSTGRES_USER: "nu"
+      POSTGRES_PASSWORD: "nupassword"
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
+      interval: 10s
+      retries: 10
+    volumes:
+      - nussknacker_designer_data:/var/lib/postgresql/data
+
+  ### Nussknacker metrics-related services:
+
+  grafana:
+    image: grafana/grafana:7.5.17
+    restart: unless-stopped
+    environment:
+      GF_AUTH_ANONYMOUS_ENABLED: "true"
+      GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s:/grafana"
+      GF_SECURITY_ALLOW_EMBEDDING: "true"
+    depends_on:
+      influxdb:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z localhost 3000"]
+      interval: 10s
+      retries: 10
+    volumes:
+      - ./grafana:/etc/grafana/provisioning
```

Comment:
./grafana/datasources:/etc/grafana/provisioning

Reply:
What is the advantage of this approaach?